lib/file_store/s3_store.rb

```diff
@@ -79,7 +79,13 @@ def copy_file(url, source, destination)
     def has_been_uploaded?(url)
       return false if url.blank?
 
-      parsed_url = URI.parse(url)
+      parsed_url = nil
```

Comment:
Nitpick, but you don't actually need this line, because `begin` doesn't open a new scope.